src/components/RoutingButton.js

```diff
@@ -1,10 +1,10 @@
-import Button from './Button';
+import BasicButtons from './Button';
 import AppContext from './AppContext';
 import { useContext } from 'react';
 
 function RoutingButton(props) {
     let [context, setContext] = useContext(AppContext);
-    return <Button text={props.text} onClick={() => setContext({...context, route: props.routeTarget })} />
+    return <BasicButtons className='module__item btn-nav' text={props.text} onClick={() => setContext({...context, route: props.routeTarget })} />
```

Comment:
Where does `module__item` come from? Do we have the option of bringing that class name in line with our otherwise kebab-cased classes?

Reply:
Now using Kebab case for classes - altered `module_item` to `module-item`